docs/src/browsers.md

```diff
@@ -82,9 +82,27 @@ class Program
 }
 ```
 
-:::note
-Playwright bundles a recent Chromium build, but not Google Chrome or Microsoft Edge browsers - these should be installed manually before use.
-:::
+Note that Playwright bundles a recent Chromium build, but not Google Chrome or Microsoft Edge browsers - these should be installed manually before use. You can use Playwright command line interface to install the browsers:
+
+```bash js
+# Install Chrome stable
```

Comment:
Looks similar to https://github.com/microsoft/playwright/blob/master/docs/src/cli.md#install-browsers which got recently added. Might make sense to dedup/reference?

Reply:
Dunno, having it inline is convenient but let's keep a link for now.